artemis/src/main/java/tech/pegasys/artemis/BeaconNodeCommand.java

```diff
@@ -78,10 +79,14 @@ public Integer call() {
                     node.stop();
                   }));
       return 0;
+    } catch (DatabaseStorageException ex) {
+      System.err.println(ex.getMessage());
+      System.exit(1);
```

Comment:
This should just return 1 not call `System.exit`.  PicoCLI treats the returned value as the exit code.

But I'm somewhat afraid you're going to tell me it doesn't actually exit...

Reply:
return 1 hangs and doesnt exit :)